File: Python/card.py
from payment import Payment

class Card(Payment):
    id = int
    number = int
    cvu = int
    date = str

    def __init__(self, id, number, cvu, date):
        super().__init__(id)
        self.number = number
        self.cvu = cvu
        self.date = date

File: Java/Card.java
package Java;

public class Card extends Payment {
    Integer id;
    Integer number;
    Integer cvu;
    String date;

    public Card(Integer id, Integer number, Integer cvu, String date){
        super(id);
        this.number = number;
        this.cvu = cvu;
        this.date = date;
    }
}